client/state/plugins/wporg/selectors.js

```diff
@@ -1,12 +1,13 @@
 /** @format */
-const getPlugin = function( state, pluginSlug ) {
+
+export const getPlugin = function( state, pluginSlug ) {
 	if ( ! state || ! state[ pluginSlug ] ) {
 		return null;
 	}
 	return Object.assign( {}, state[ pluginSlug ] );
 };
 
-const isFetching = function( state, pluginSlug ) {
+export const isFetching = function( state, pluginSlug ) {
```

Comment:
All three exported selectors could use renaming to a more unique name. I can get over `fetchPluginData` from the other PR, but `isFetching` is much worse. Would you mind renaming them to something that could be eventually exported from the global `state/selectors` module?

Reply:
how about we leave that for if and when we move them into the global space? they are already namespaced by module path and we can always rename them on import.

I don't mind, but I also think it means expanding this PR quite a bit since the existing code was already doing `import { isFetching } from '…'`

Reply:
OK, I now learned that they are not "real" selectors anyway: they don't select from the global Redux state, but rather from a custom `wporgPlugins` object. Some more refactoring is needed to standardize this code.